update_employees.py

```diff
@@ -344,27 +362,6 @@ def set_passwords(records, password_field):
     return
 
 
-def remove_empty_emails(payload, email_field, name_field):
```

Comment:
as far as I can tell this isn't possible because we assign emails to new users who don't have them.